tests/integration/test_grpc_protocol/test.py

```diff
@@ -369,47 +369,33 @@ def test_progress():
         "SELECT number, sleep(0.31) FROM numbers(8) SETTINGS max_block_size=2, interactive_delay=100000",
         stream_output=True,
     )
-    results = list(results)
-    for result in results:
-        result.time_zone = ""
-        result.query_id = ""
-    # print(results)
-
-    # Note: We can't convert those messages to string like `results = str(results)` and then compare it as a string
-    # because str() can serialize a protobuf message with any order of fields.
-    expected_results = [
-        clickhouse_grpc_pb2.Result(
-            output_format="TabSeparated",
-            progress=clickhouse_grpc_pb2.Progress(
-                read_rows=2, read_bytes=16, total_rows_to_read=8
-            ),
-        ),
-        clickhouse_grpc_pb2.Result(output=b"0\t0\n1\t0\n"),
-        clickhouse_grpc_pb2.Result(
-            progress=clickhouse_grpc_pb2.Progress(read_rows=2, read_bytes=16)
-        ),
-        clickhouse_grpc_pb2.Result(output=b"2\t0\n3\t0\n"),
-        clickhouse_grpc_pb2.Result(
-            progress=clickhouse_grpc_pb2.Progress(read_rows=2, read_bytes=16)
-        ),
-        clickhouse_grpc_pb2.Result(output=b"4\t0\n5\t0\n"),
-        clickhouse_grpc_pb2.Result(
-            progress=clickhouse_grpc_pb2.Progress(read_rows=2, read_bytes=16)
-        ),
-        clickhouse_grpc_pb2.Result(output=b"6\t0\n7\t0\n"),
-        clickhouse_grpc_pb2.Result(
-            stats=clickhouse_grpc_pb2.Stats(
-                rows=8,
-                blocks=4,
-                allocated_bytes=1092,
-            )
-        ),
+
+    # Note: We can't compare results using a statement like `assert results == expected_results`
+    # because `results` can come in slightly different order.
+    # So we compare `outputs` and `progresses` separately and not `results` as a whole.
+
+    outputs = [i.output for i in results if i.output]
+    progresses = [i.progress for i in results if i.HasField("progress")]
+
+    # print(outputs)
```

Comment:
Debug prints?

Reply:
yes

Reply:
Pls remove :slightly_smiling_face:

Reply:
why?

Reply:
```
➜  integration git:(master) ✗ pwd
/home/thevar1able/src/clickhouse/tests/integration
➜  integration git:(master) ✗ rg "^\s+#" | grep "print(" | wc -l
22
```

It is not hard to add them if necessary, and I have a feeling that it is cleaner to have code without commented out code lines. I see it mostly in your tests, feel free to ignore this need if you disagree with me.

Reply:
It's not a big deal to remove those comments. I believe those prints are good because they give kind of hints where to look in a test if something goes wrong.